test/activate_set.jl

```diff
@@ -39,4 +39,16 @@
             Pkg.activate(orig_project_toml_path)
         end
     end
+
+    # See #26
```

Comment:
I prefer full URLs for comments so I can click them to go the other code
```suggestion
    # See https://github.com/JuliaTesting/TestEnv.jl/issues/26
```